cmd/lifecycle/restorer.go

```diff
@@ -124,12 +124,12 @@ func (r *restoreCmd) Exec() error {
 			if err != nil {
 				return cmd.FailErr(err, "read target data from run image")
 			}
-			ref, err := digestReference(analyzedMD.RunImage.Reference, runImage)
+			digestRef, err := digestReference(analyzedMD.RunImage.Reference, runImage)
```

Comment:
`runImage.Identifier()` might do here

Reply:
On reading the code again, no - a sparse image identifier includes the file path